### DISCUSSION TITLE
Prebuilt versions?

### DISCUSSION THREAD

**Discussion by Kippykip** [tenacityteam/tenacity-legacy#262] [Q&A] [answered]
For this version to take off, there absolutely needs to be prebuilt versions of Tenacity for Windows and Linux.
A lot of peeps find having to compile software to use it a massive hassle, especially when they're working full day jobs or have a large quantity of software already distributed as source only.

Is there any consideration of hosting prebuilt installers in a .deb or .exe format under the releases tab?

**Comment by purplesyringa** [comment, marked as answer]
Yes, we will do that once we are ready to release the first version of Tenacity.

**Comment by ajayyy** [comment]
For now, you can use https://nightly.link/tenacityteam/tenacity/workflows/cmake_build/master to get the builds from GitHub actions, but these don't have all branding removed yet, so shouldn't be publicised too much

**Comment by emabrey** [comment]
We are absolutely planning on making pre-built binaries. We have to be ready for release first!
